disaster_recovery/restore/disaster_recovery.py

```diff
@@ -0,0 +1,621 @@
+        )
+        self.StorageEncrypted = response['DBClusters'][0]['StorageEncrypted']
+        self.KmsKeyId = response['DBClusters'][0]['KmsKeyId']
+        self.Capacity = None if 'Capacity' not in response['DBClusters'][0] else response['DBClusters'][0]['Capacity']
+        self.EngineMode = response['DBClusters'][0]['EngineMode']
+        self.DeletionProtection = response['DBClusters'][0]['DeletionProtection']
+        self.TagList = response['DBClusters'][0]['TagList']
+        self.AutoMinorVersionUpgrade = response['DBClusters'][0]['AutoMinorVersionUpgrade']
+        self.HttpEndpointEnabled = response['DBClusters'][0]["HttpEndpointEnabled"]
+
+    def restore_from_snapshot(self):
+        if self.same_target:
+            self.db_cluster_identifier_target = f"{self.db_cluster_identifier_source}-temp"
+
+        self.restore_cluster_ss()
+        if self.EngineMode != 'serverless':
+            self.create_db_instances()
+
+        if self.same_target:
+            self.overwrite_existing_cluster()
+
+    def restore_cluster_ss(self):
+        if self.EngineMode == 'serverless':
+            response = self.client.restore_db_cluster_from_snapshot(
+                DBClusterIdentifier=self.db_cluster_identifier_target,
+                SnapshotIdentifier=self.SnapshotIdentifier,
+                Engine=self.Engine,
+                EngineVersion=self.EngineVersion,
+                VpcSecurityGroupIds=self.VpcSecurityGroups,
+                DBSubnetGroupName=self.DBSubnetGroup,
+                DeletionProtection=self.DeletionProtection,
+                EngineMode=self.EngineMode,
+                KmsKeyId=self.KmsKeyId,
+                Tags=self.TagList,
+                HttpEndpointEnabled=self.HttpEndpointEnabled
+            )
+        else:
+            response = self.client.restore_db_cluster_from_snapshot(
+                DBClusterIdentifier=self.db_cluster_identifier_target,
+                SnapshotIdentifier=self.SnapshotIdentifier,
+                Engine=self.Engine,
+                EngineVersion=self.EngineVersion,
+                VpcSecurityGroupIds=self.VpcSecurityGroups,
+                DBSubnetGroupName=self.DBSubnetGroup,
+                EnableCloudwatchLogsExports=['postgresql'],
+                DeletionProtection=self.DeletionProtection,
+                EngineMode=self.EngineMode,
+                KmsKeyId=self.KmsKeyId,
+                Tags=self.TagList,
+                HttpEndpointEnabled=self.HttpEndpointEnabled
+            )
+
+        self.command_response('Restore from snapshot', response)
+
+        self.wait_on_cluster_available()
+
+    def restore_to_point_in_time(self):
+        if self.same_target:
+            self.db_cluster_identifier_target = f"{self.db_cluster_identifier_source}-temp"
+
+        self.restore_cluster_pitr()
+
+        if self.same_target:
+            self.overwrite_existing_cluster()
+
+    def restore_cluster_pitr(self):
```

Comment:
Same as `ss` - could we get this expanded to `point_in_time_xxxx` (not sure what the r stands for 😅 )

Reply:
:)